src/optimizer/projection_push_down_optimizer.cpp

```diff
@@ -266,6 +267,28 @@ void ProjectionPushDownOptimizer::visitTableFunctionCall(LogicalOperator* op) {
     tableFunctionCall.setColumnSkips(std::move(columnSkips));
 }
 
+void ProjectionPushDownOptimizer::visitScanNodeTable(LogicalOperator* op) {
+    clearScanNodeTableProperties(op);
+}
+
+bool ProjectionPushDownOptimizer::clearScanNodeTableProperties(planner::LogicalOperator* op) const {
+    auto& scanNodeTable = op->cast<LogicalScanNodeTable>();
+    if (scanNodeTable.getProperties().empty()) {
+        return false;
+    }
+    bool hasPropertiesInUse = false;
```

Comment:
First I wanna understand on which query this optimization will take effect so that we can add test & benchmark.

Second, I think we can do better than `clearProperty`. Scan operator can scan a subset of its current properties.